plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorUtils.java

```diff
@@ -544,4 +546,28 @@ private static void notifyPrefs(@NotNull DBPPreferenceStore prefStore, boolean n
             newServicesEnabled && markWordForSelectionEnabled
         );
     }
+    
+    /**
+     * Returns type id of the driver of data source container, associated with SQLEditor
+     * Returns null if editor is not instance fo SQLEditor or data source container is null
```

Comment:
This will be merged into a single line.

```suggestion
     * Returns type id of the driver of data source container, associated with SQLEditor, or {@code null} if editor is not instance fo SQLEditor or data source container is null
```